### ISSUE TITLE
Error with xgboost.cv seed parameter

### ISSUE THREAD

**Issue by praneesh12** [dmlc/xgboost#4180]
#### I'm having trouble working with xgboost.cv(). Here is the code snippet of xgboost.cv(), seed parameter has a default value but still shows an error even when I don't specify anything. I'm using 0.81 version of xgboost with python 3.7.
------------------------------------------------------------------------------------------------
clf_xgb = xgb.XGBClassifier()
hyperparameters = clf_xgb.get_params()
d_train = xgb.DMatrix(X_train_scaled, y_train)
xgb.cv(hyperparameters, d_train)
------------------------------------------------------------------------------------------------
Error message
<img width="1042" alt="screen shot 2019-02-25 at 4 28 49 pm" src="https://user-images.githubusercontent.com/25232987/53369972-736d8100-391a-11e9-8b20-b9c13f4e37fd.png">

**Comment by hcho3** [comment]
@praneesh12 You should use [`get_xgb_params()`](https://xgboost.readthedocs.io/en/latest/python/python_api.html#xgboost.XGBRegressor.get_xgb_params) instead of `get_params()`.
